sklearn/gaussian_process/_gpc.py

```diff
@@ -555,6 +555,10 @@ def optimizer(obj_func, initial_theta, bounds):
 
     Attributes
     ----------
+    base_estimator_ : ``_BinaryGaussianProcessClassifierLaplace``
```

Comment:
The type description seems a bit overcomplicated. Maybe use `Estimator` instead?

Reply:
Yeah, we will do. We had doubts about the naming :)